static/js/services/lineage-model-generator.js

```diff
@@ -20,7 +20,8 @@ angular.module('inboxServices').factory('LineageModelGenerator',
         .query('medic-client/docs_by_id_lineage', {
           startkey: [ id ],
           endkey: [ id, {} ],
-          include_docs: true
+          include_docs: true,
+          attachments: true,
```

Comment:
Yeah, I'm not sure about this. Reports have the XML attached so doing this will mean each row will fetch the raw XML attachment and then not use it. I think the right way is probably to fetch the attachment specifically on render. That way when you're just fetching the summaries for the LHS list you don't get all the images. The images are lazy loaded only when the report is selected.